flake.nix

```diff
@@ -10,10 +10,15 @@
 
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
 
+    php82 = {
+      url = "https://downloads.php.net/~pierrick/php-8.2.0alpha2.tar.xz";
+      flake = false;
+    };
+
```

Comment:
Could you also squash the [Remove flake input and use fetchurl.](https://github.com/fossar/nix-phps/pull/118/commits/9c3a2cce89f8f471b461fec6b923f5f99b114c9a) into the parent commit? I do not think we need this in history.

Reply:
Yep, will do it.